lib/formatters/FormatterFactory.js

```diff
@@ -0,0 +1,21 @@
+const SimpleFormatter = require('./SimpleFormatter');
+const ExtendedFormatter = require('./ExtendedFormatter');
+const JsonFormatter = require('./JsonFormatter');
+
+const create = (type = '', color = false, appendDate = false, requestType = '') => {
+  switch (type) {
+    case 'simple': {
+      return new SimpleFormatter(color, appendDate, requestType);
+    }
+    case 'extended': {
+      return new ExtendedFormatter(color, appendDate, requestType);
+    }
+    case 'json': {
+      return new JsonFormatter(color, appendDate, requestType);
+    }
+    default:
+      return null;
```

Comment:
Nit: Maybe throw an error to early catch problematic setup?

Reply:
Added "MissingFormatter" to inform for the wrong parameter without breaking other formatters